nengo_spinnaker/builder/node.py

```diff
@@ -118,11 +118,7 @@ def build_node(self, model, node):
             # If the Node is a passthrough Node then create a new filter object
             # for it.  We might be requested to use a fixed number of cores or
             # chips, we extract that information from the config.
```

Comment:
<nitpick>Should this comment be updated to reflect the fact that fixed numbers of cores and chips aren't specified here</nitpick>